projects/plugins/wpcomsh/feature-plugins/post-list.php

```diff
@@ -16,34 +16,10 @@
  */
 function wpcomsh_post_list_init() {
 	add_filter( 'jetpack_block_editor_republicize_feature', '__return_true' );
-	wpcomsh_maybe_enable_share_action();
```

Comment:
Will this change cause issues on Atomic when they receive the wpcomsh change today but Jetpack change on Monday?

Reply:
Yes, my apprehension is correct. I tested this change with wpcomsh from this branch and Jetpack RC and the Share button is gone.

<img width="472" height="97" alt="image" src="https://github.com/user-attachments/assets/8c9fddab-4cf6-4c83-8c70-d656179f75b4" />

So, it means that we may need to hold off merging this till Monday morning.

Reply:
Otherwise, we can extract the wpcomsh change into a separate PR and merge that after the next Atomic deployment. That will unblock this PR.

Reply:
I think it's fine to wait until Monday, it's not urgent to ship, and not likely to get conflicts either

Reply:
Sounds good. Thanks